### PR TITLE
update examples and intro

### PR DESCRIPTION
**Pull request by logan-anderson**
This is a draft because it depends on https://github.com/tinacms/tinacms.org/pull/1293 to be merged first

### General Contributing:

* [ ] Have you followed the guidelines in our [Contributing document](https://github.com/tinacms/tinacms.org/blob/master/CONTRIBUTING.md)?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### All New Content Submissions: (To be confirmed by reviewer)

* [ ] Title is short & specific
* [ ] Headers are logically ordered & consistent
* [ ] Purpose of document is explained in the first paragraph
* [ ] Procedures are tested and work
* [ ] Any technical concepts are explained or linked to
* [ ] Document follows structure from templates
* [ ] All links work
* [ ] Spelling and grammer checker has been run
* [ ] Graphics and images are clear and useful
* [ ] Any prerequisites and next steps are defined.

### CODE REVIEW

File: content/docs/schema.md [hunk 1]
```diff
@@ -173,10 +182,36 @@ This will result in a resolvable node in your GraphQL structure (Don't worry, we
...
 ```
 
+
 The resulting field in your TinaCMS form will be a `select` field, whose `options` are all of the documents in the referenced collections.
 
+## Customizing fields with the `UI` property
+
+<!-- TODO add links when https://github.com/tinacms/tinacms.org/pull/1293/files is merged -->
+
+Fields can customized to allow a better experience for the content editors. The `ui` property on each field can enable: [custom frontend validation functions](/docs/extending-tina/validation/) which does not allow editors to submit values that are invalid, [formate and parse](/docs/extending-tina/format-and-parse/) input to allow customization of what is saved to the document and what is displayed to the user, and allow [custom field components](/docs/extending-tina/custom-field-components/) to display any input imaginable in the CMS.
```
Comment (jeffsee55) [inline content/docs/schema.md:192]:
```suggestion
Fields can be customized to provide a better experience for content editors. The `ui` property on each field can enable: [custom frontend validation functions](/docs/extending-tina/validation/) which does not allow editors to submit values that are invalid, [formate and parse](/docs/extending-tina/format-and-parse/) functions to allow customization of what is saved to the document and what is displayed to the user, and allow [custom field components](/docs/extending-tina/custom-field-components/) to display any input imaginable in the CMS. 
```

File: content/docs/schema.md [hunk 2]
```diff
@@ -173,10 +182,36 @@ This will result in a resolvable node in your GraphQL structure (Don't worry, we
...
+        return 'The title field must be more then 5 characters'
+      }
+    }
+  }
+}
+```
 ## Summary
 
-- Your content is modeled in the .tina/schema.ts of your repo
+- Your content is modeled in the .tina/schema.{ts,js,tsx} of your repo
```
Comment (jeffsee55) [inline content/docs/schema.md:215]:
```suggestion
- Your content is modeled in the `.tina/schema.{ts,js,tsx}` of your repo
```

**Review by Enigmatical — APPROVED** [review]
Pending @jeffsee55 's grammatical changes, I'm good with this 👍

**Comment by logan-anderson** [comment]
Fixed them @Enigmatical

**Review by ashleyjanedoiron — APPROVED** [review]

**Review by perkinsjr — APPROVED** [review]

**Pull request merged**